package.json

```diff
@@ -30,6 +30,8 @@
     "@khanacademy/eslint-config": "^3.0.1",
     "@khanacademy/eslint-plugin": "^2.1.1",
     "@khanacademy/wonder-blocks-button": "6.2.5",
+    "@khanacademy/wonder-blocks-i18n": "^2.0.2",
+    "@khanacademy/wonder-blocks-layout": "^2.0.27",
```

Comment:
These are used by our Storybook `preview.tsx` and so Vite needed them to be added in our root package.